CRISPResso2/CRISPRessoCORE.py

```diff
@@ -4861,6 +4867,7 @@ def get_scaffold_len(row, scaffold_start_loc, scaffold_seq):
                         'sgRNA_intervals': new_sgRNA_intervals,
                         'sgRNA_names': sgRNA_names,
                         'sgRNA_mismatches': sgRNA_mismatches,
+                        'sgRNA_sequences': [sgRNA],
```

Comment:
may need to use sgRNA_sequences here, right?

Reply:
I believe the `sgRNA` is referring to this line https://github.com/pinellolab/CRISPResso2/pull/432/files/fa5a18d8bbffe5983bbbed0e42f539703b90e44d#diff-6043decbacd7b8775dc4013839dc08bcb168ec37f1031a60e8ee89291d23c06eR4836